### ISSUE TITLE
alt text should be a description of the image

### ISSUE THREAD

**Issue by martmatwarne** [Owen045/UdacityProjects#6]
The alt text is what will display if the image fails to load, as well as what screen readers will read out and what Google will read. Therefore it should be a description of the photo like "Dwarf Date Palm" rather than "image not found"
